src/styleguide/header-light.html

```diff
@@ -0,0 +1,101 @@
+<!doctype html>
```

Comment:
Sharp edges on automating it so far: layouts, partials not included (and not yet extracted from the website), and required contexts to generate the right HTML.